app/decorators/has_hint_option.rb

```diff
@@ -4,6 +4,12 @@ def options
   end
 
   def hint
-    nil
+    "Must fulfill: #{hint_message}" if hint_message
+  end
+
+  private
+
+  def hint_message
+    Array(object.validators[property]).map(&:message)[0]
```

Comment:
Why are you only pulling the first validator's message?  If we have multiple validators for a field, we might need then all to show up.